dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -144,13 +151,13 @@ CDefaultContextMenu::CDefaultContextMenu() :
     m_pidlFolder(NULL),
     m_bGroupPolicyActive(0),
     m_iIdSHEFirst(0),
-    m_iIdSHELast(0),
+    m_iIdSHEEnd(0),
```

Comment:
Why not keeping the "Last" qualifier?

Reply:
since, at least to me, "last" sounds like it's the id of the last command included even though it's not (e.g. first 5 last 8 includes 5, 6, 7, but not 8), while "end" is already used in the "smaller than this" sense e.g. in STL. I can change it back though if you think it's clear enough already!

(at some point I was convinced that this is where the one-off is coming from... it wasn't though :))